DataFormats/L1TMuon/interface/EMTFOutput.h

```diff
@@ -70,7 +71,8 @@ namespace l1t {
     emtf::EventTrailer GetEventTrailer()    { return EventTrailer;  };
     emtf::MTF7Trailer GetMTF7Trailer()      { return MTF7Trailer;   };
     emtf::AMC13Trailer GetAMC13Trailer()    { return AMC13Trailer;  };
-    const uint64_t Dataword() const     { return dataword; };
+    const int Format_Errors() const { return format_errors; };
+    const uint64_t Dataword() const { return dataword; };
```

Comment:
@mulhearn 
These two methods (Format_Errors() and Dataword()) are both missing in the 81X version of this file (#13768), while all methods from line 54 to line 73 were const in the 81X incarnation, but not here. Is all that wanted, or any of the two PR's for 81X or 80X should be fixed, instead?